static/temp/recipe_preferences.json

```diff
@@ -0,0 +1,32 @@
+{
+  "setYourPreferences": {
+    "all": [
+      { "name": "Dietary" },
+      { "name": "Allergy" },
+      { "name": "Meal type" },
+      { "name": "Cooking time" }
+    ],
+    "preference": [
+      {
+        "filterName": "Dietary",
```

Comment:
replace `filterName` to `name`. This change should be for everywhere where `filterName` exists.

Reply:
done

Reply:
done